WebExtension/bridge.js

```diff
@@ -26,7 +26,8 @@ try {
 	var storage = browser.storage.local;
 	var storage_loaded = false;
 	var framework_version = getVersion();
-	var storage_used = -1;
+	// Default to unlimited storage since we have that pref
```

Comment:
By pref I meant permission, i.e. `unlimitedStorage` in `manifest.json`, which technically isn't officially supported by Firefox so really this comment is just trying to add a good-sounding explanation where the truth is "default to 0 bytes in use because otherwise XKit will be sad"

I'm just going to delete the comment because defaulting to 0 makes more sense anyway